framework/core/js/src/forum/components/NotificationList.js

```diff
@@ -33,6 +33,10 @@ export default class NotificationList extends Component {
     const items = new ItemList();
     const state = this.attrs.state;
 
+    if (!state.hasItems()) {
+      return items;
+    }
+
```

Comment:
actually, it would probably be better to wrap the two available items in the conditional, to not completely prevent adding for example a link to notification preferences as an item from an extension?